_posts/2020_NeurIPS_speakers/2019-06-05-Andres.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Andres Muñoz Medina
```

Comment:
Only one post template for the speaker page should be carried over for reference for this workshop, please delete the duplicates.